docs/Stdlib/python/Tutorials/machine_learning.md

```diff
@@ -21,38 +23,51 @@ to_be_predicted_table = Table({
 sup_dataset = SupervisedDataset(table, target_column="target")
 ```
 
-SupervisedDatasets are used in safe-DS to train supervised machine learning models, because they keep track of the target
-vector. A SupervisedDataset can be created from a table and specifying the target vector in the table.
+[SupervisedDatasets](../data/SupervisedDataset.md) are used in safe-DS to train supervised machine learning models
+(e.g. [RandomForest](../classification/RandomForest.md) for classification and
+[LinearRegression](../regression/LinearRegression.md) as a regression model), because they keep track of the target
+vector. A [SupervisedDataset](../data/SupervisedDataset.md) can be created from a [Table](../data/Table.md) and
+specifying the target vector in the table.
+
+## Create and train model
 
 In this code example, we want to predict the sum of a row. The `table` contains the target vector we want to
 train with (the sum of the rows). The `to_predicted_table` is the table we want to make predictions with, so it
 does not contain a target vector.
 
-In order to train the `LinearRegression`-model we need to make the following calls in safe-DS.
+In order to train the [LinearRegression](../regression/LinearRegression.md)-model we need to make the following calls
+in safe-DS:
 
 ```python
 linear_reg_model = LinearRegression()
 linear_reg_model.fit(sup_dataset)
 ```
 
-As we can see, a `LinearRegression`-object is created.
+As we can see, a [LinearRegression](../regression/LinearRegression.md)-object is created.
 
 In safe-DS machine learning models are separated in different classes where the different fit and predictions methods
 are implemented for the given machine learning model.
 
-So in order to train a linear regression model we create a `LinearRegression`-object and call then the `.fit()`
--method on this object. Now the `linear_reg_model` is a fitted linear regression model, and we can call
-the `predict(dataset = SupervisedDataset)`-method on this model.
+## Predicting new values
+
+So in order to train a linear regression model we create a [LinearRegression](../regression/LinearRegression.md)-object
+and call then the [`.fit()`](../regression/LinearRegression.md#safe_ds.regression._linear_regression.LinearRegression.fit)
```

Comment:
I didn't test it yet but writing this should also work and doesn't depend on the file structure of the documentation:

```suggestion
and call then the [`.fit()`](safe_ds.regression._linear_regression.LinearRegression.fit)
```

Reply:
All relative paths are now replaced with python class names or method names